Topics/Development_Process/Github_Pages.md

```diff
@@ -0,0 +1,91 @@
+# GitHub Pages
+
+If you have ever created a "site" before using HTML, maybe with CSS and perhaps Javascript, and wondered how you could host it online through a link, here's your solution.
+
+From the [GitHub docs](https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages):
+
+> GitHub Pages is a static site hosting service that takes HTML, CSS, and JavaScript files straight from a repository on GitHub, optionally runs the files through a build process, and publishes a website.
+
+In other words, it is a **free static website host.** It's reliable, your pages will remain up forever (unless GitHub changes this), and can handle a reasonable amount of bandwidth (100GB/mo), given you aren't serving images or large files.
+
+**Note:** Free for public repositories. You need GitHub pro if you want to do this for private repositories. The deployed "page" will be public but your repo will remain private. Students should be able to get Pro for free.
```

Comment:
I doesn't look like "Github pro" is the official name of Github's paid subscription plan. It might be more helpful to change this to be more descriptive, and maybe to add a link to a page on Github's website that displays the subscription plans.